packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```diff
@@ -360,18 +360,11 @@
             "description": "The dashboard name. Must match the 'name' attribute in the dashboard file."
           },
           "condition": {
-            "type": ["null", "string"],
```

Comment:
`condition` is the same for all these assets, so I pulled it into schema definition down below like `onFailure` and `overwriteOnUpgrade`.